portafly/portafly_api.yml

```
@@ -85,46 +237,114 @@ components:
-                apps_count: 5
-              - id: 40
-                created_at: 2018-02-11T09:30Z
-                updated_at: 2018-02-11T09:30Z
-                state: pending
-                org_name: B.P.R.D
-                admin_name: Dr. Thomas Manning
-                apps_count: 10
-              - id: 41
-                created_at: 2018-02-12T09:30Z
-                updated_at: 2018-02-12T09:30Z
-                state: approved
-                org_name: MomCorp
-                admin_name: Mom
-                apps_count: 2
-              - id: 42
-                created_at: 2018-02-12T09:30Z
-                updated_at: 2018-02-12T09:30Z
-                state: approved
-                org_name: Sirius Cybernetics
-                admin_name: Admin
-                apps_count: 3
-              - id: 43
-                created_at: 2018-03-10T09:30Z
-                updated_at: 2018-03-10T09:30Z
-                state: approved
-                org_name: Los Pollos Hermanos
-                admin_name: Gustavo Fring
-                apps_count: 4
+                - id: 22
+                  created_at: 2018-02-10T09:30Z
+                  updated_at: 2018-02-10T09:30Z
+                  state: approved
+                  org_name: Capsule Corp
+                  admin_name: Dr. Brief
+                  apps_count: 4
+                - id: 72
+                  created_at: 2018-02-10T09:30Z
+                  updated_at: 2018-02-10T09:30Z
+                  state: approved
+                  org_name: FOXHOUND
+                  admin_name: Big Boss
+                  apps_count: 5
+                - id: 40
+                  created_at: 2018-02-11T09:30Z
+                  updated_at: 2018-02-11T09:30Z
+                  state: pending
+                  org_name: B.P.R.D
+                  admin_name: Dr. Thomas Manning
+                  apps_count: 10
+                - id: 41
+                  created_at: 2018-02-12T09:30Z
+                  updated_at: 2018-02-12T09:30Z
+                  state: approved
+                  org_name: MomCorp
+                  admin_name: Mom
+                  apps_count: 2
+                - id: 42
+                  created_at: 2018-02-12T09:30Z
+                  updated_at: 2018-02-12T09:30Z
+                  state: approved
+                  org_name: Sirius Cybernetics
+                  admin_name: Admin
+                  apps_count: 3
+                - id: 43
+                  created_at: 2018-03-10T09:30Z
+                  updated_at: 2018-03-10T09:30Z
+                  state: approved
+                  org_name: Los Pollos Hermanos
+                  admin_name: Gustavo Fring
+                  apps_count: 4
       description: Developer accountlist
+    userList:
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              $ref: '#/components/schemas/User'
+          examples:
```

When there's only one exampl I would use `example` instead.
```suggestion
          example:
```

---

But in this case is an array

---

`examples` is to add different examples and name then: accountUserListExample, emptyAccountUserListExample, longAccountUserListExample...
`example` is meant to be used with one single, unnamed example, no matter its type (array, object)
https://github.com/3scale/porta/pull/2005/files#diff-de213b3da63e2294b43e510a86210a83R168-R195